packages/cli/src/databases/migrations/sqlite/1669730543736-addMfaColumns.ts

```diff
@@ -0,0 +1,76 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { getTablePrefix, logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+export class AddMfaColumns1669730543736 implements MigrationInterface {
```

Comment:
this should be renamed to be chronologically the latest migration.
done in https://github.com/n8n-io/n8n/commit/824349de114cfd84ef2ec293be2090cba2a5d338 
please feel free to cherry-pick.